CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.5.0] - 2017-11-16
+
+### Summary
+This is a feature and bug-fix release
+
+### Features
+- Add ability to manage the `stringify_facts` setting ([MODULES-5953](https://tickets.puppetlabs.com/browse/MODULES-5953))
+- Upgrades to Puppet 5 are now supported on RPM-based platform ([MODULES-5633](https://tickets.puppetlabs.com/browse/MODULES-5633))
+- Debian 9 is now supported
```

Comment:
https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/260 was the change, it was a PE-specific issue.